kfb19/git-training
# Adding in a for loop

Changes made: 

- For loop added to print numbers 0, 1 and 2

How to test: 

- Run code using python main.py, you should see "hello world" printed, and then the numbers 0, 1, 2 printed each on a line
